src/systemctl/systemctl.c

```diff
@@ -5491,6 +5502,8 @@ static int show_one(
                 .tasks_max = (uint64_t) -1,
                 .ip_ingress_bytes = (uint64_t) -1,
                 .ip_egress_bytes = (uint64_t) -1,
+                .io_read_bytes = UINT64_MAX,
```

Comment:
Hmm, any particular to use `(uint64_t) -1` above and `UINT64_MAX` here?

Reply:
Well, no not really. In newer code I tend to write UINT64_MAX more often than (uint64_t) -1, since the latter involves an explicit cast that only works on 2s complement systems, while the former doesn't care about casts or 2s complement or anything like that, and works on all ANSI C. Of course we don't support systemd on any non-2s-complement system (and I am pretty sure we should refuse such patches if people would start sending them), but I still think on a philosophical, semantical level it makes more sense to explicitly say "use the maximum uint64_t value here" instead of "uh, so, please cast this completely weird value into uint64_t which is a type which can't represent this value but I know what I am doing, so the result is what I want it to be on the machines I care about".

But yeah, this totally doesn't matter at all, given the everyone hacking C knows what `(uint64_t) -1` means and is supposed to mean anyway, but it still feels a bit nicer...

If you value consistency over such philosophical thoughts then I'd be happy to change this to (uint64_t) -1 again though ;-)

Reply:
No, I'd rather change the other ones to UINT64_MAX, but let's do that in a later patch.